Scripts/ReinstallGVFS.bat

```diff
@@ -1,6 +1,9 @@
+@ECHO OFF

```

Comment:
This is another script that is intended for build machines only.

Maybe outside the scope of this PR, but I think the following would be a very nice additional cleanup:
* Move all scripts in the `Scripts` folder into `Scripts/Windows`
* Further separate the scripts into those intended to be run on dev machines and those that are for build machines only. At a first glance, all of the scripts you have aliased (except for scorch) could go in `Scripts/Windows/Dev` (or just leave them in `Scripts/Windows` to save some letters) and many of the rest could be moved to `Scripts/Windows/BuildAgents`

Thoughts?

Reply:
I like that idea. Let me see what I can do...

Reply:
If we go moving all of this around now, before we are on yaml pipelines, we will have to modify the build definitions. Let's hold off on this cleanup until our pipelines are versioned with the code.

Reply:
Yep doing it once we move to yaml is a good idea.

Reply:
Deferring for now.